docs/src/management-commands.rst

```diff
@@ -198,6 +198,23 @@ Reset MT budget of regions whose renewal month is the current month::
 * ``--force``: Allow to reset the budget even if it's not the first day of the month
 
 
+``update_link_text``
+~~~~~~~~~~~~~~~~~~~~~~
+
+Update ALL link text of links of the given URL:
+
+    integreat-cms-cli update_link_text [--target_url TARGET_URL] [--new_link_text NEW_LINK_TEXT] [--username USERNAME]
```

Comment:
It seems the arguments use hyphens, not underscores. Also, please make the important arguments required and mark them as such
```suggestion
    integreat-cms-cli update_link_text --target-url TARGET_URL --new-link-text NEW_LINK_TEXT [--username USERNAME]
```